docs/getting-started/tutorial-load-dataset.asciidoc

```diff
@@ -78,7 +78,7 @@ curl -H 'Content-Type: application/json' -XPUT http://localhost:9200/shakespeare
 
 This mapping specifies the following qualities for the data set:
 
-* The _speaker_ field is a string that isn't analyzed. The string in this field is treated as a single unit, even if
+* The _speaker_ field is a keyword that isn't analyzed. The text in this field is treated as a single unit, even if
```

Comment:
thoughts on changing this to `...field that isn't analyzed`?  I think the not analyzed portion overlaps with keyword.